website/docs/procs.md

```diff
@@ -124,7 +124,7 @@ end
 ```
 
 This also happens a lot with certain Rails APIs, etc. Sorbet has direct support
-for this sort of pattern using an extra argument `T.proc` called `.bind`:
+for this sort of pattern using an extra argument on `T.proc` called `.bind`:
```

Comment:
I am not fully familiar with internal Sorbet jargon so wasn't sure if `.bind` was an _argument_ on `T.proc` or a call.